circuits/src/tests/keccak/mod.rs

```diff
@@ -464,8 +464,7 @@ impl<F: EccPrimeField> AssignedKeccakInput<F> {
         self.len().value().get_lower_32() as usize
     }
 
-    /// Returns 1 if the input uses the optional LegoSNARK commitment and
-    /// 0 otherwise.
+    /// Returns 1 if the input uses the optional commitment, and 0 otherwise.
```

Comment:
```suggestion
/// Returns 1 if the input uses the optional Pedersen commitment, and 0 otherwise.
```

Reply:
Seems to suggest removing the indentation.  Was that intentional?

Reply:
no